src/index.ts

```diff
@@ -21,12 +21,19 @@ export interface UseAsyncFetcherReturn {
   ) => Promise<T>;
 }
 
+
+const alphabet = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const generate = customAlphabet(alphabet, 12)
+const generateRequestId = () => {
+  return generate()
+}
+
 export function useAsyncFetcher(): UseAsyncFetcherReturn {
   const originalFetcher = useFetcher();
 
   const fetch = useCallback(
     async <T>(href: string): Promise<T> => {
-      const requestId = nanoid();
+      const requestId = generateRequestId();
```

Comment:
I'd be tempted to just use [`ulid()`](https://www.npmjs.com/package/ulid) for this, since they're not user-facing the alphabet etc isn't terribly important?

Reply:
I agree the alphabet isn't important. How about using `window.crypto.randomUUID()`?